components/registration/registration-controller.js

```diff
@@ -587,7 +587,8 @@ trackerCapture.controller('RegistrationController',
                 closeButtonText: 'no',
                 actionButtonText: 'yes',
                 headerText: 'cancel',
-                bodyText: 'are_you_sure_to_cancel_registration'
+                //Depending on if you are editing in dashboard or registering a new TEI you get a different cancel message.
+                bodyText: inDashboard ? 'are_you_sure_to_cancel_editing' : 'are_you_sure_to_cancel_registration'

```

Comment:
is it possible to use $scope.isInDashboard here, instead of passing in the inDashboard parameter to the function?